docker/ubuntu/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN PIPX_HOME=/opt/pipx \
          PIPX_BIN_DIR=/usr/bin \
          PIPX_MAN_DIR=/usr/share/man \
          pipx install conan
+RUN apt purge -y pipx && apt autoremove -y
```

Comment:
Don't do that, it removes Python packages which are required for `conan` to work.